### ISSUE TITLE
Add Author Profile pic in Applens

### ISSUE THREAD

**Issue by cindylovescoding** [ShekharGupta1988/AppLensV2#1258]
Add Author Profile pic in Applens

**Comment by stevenernst131** [comment]
Can someone make the case to me that this is something we should make a priority?

**Comment by puneetg1983** [comment]
I personally don't consider this as priority but Praveen suggested this  :)

**Comment by cindylovescoding** [comment]
Right.. I have put this into backlog first, and will work on this after more prioritized tasks on my hand.
